### ISSUE TITLE
Hi, I try to train the gomoku. But did not get the expected effect.

### ISSUE THREAD

**Issue by zfavourite99** [werner-duvaud/muzero-general#41]
After the release of AlphaZero, I trained Gomoku based on AlphaZero, and I didn't need much calculation to get a good model.

The code used is the code of [this](https://github.com/junxiaosong/AlphaZero_Gomoku).

I train a simple game(6x6 board and 4 chess in line will win) on cpu(i7 2.5G Hz). It only takes about 10 hours. And the code is single thread.

I train a simple game(8x8 board and 5 chess in line will win) on cpu(i7 2.5G Hz). It only takes about 50 hours. And the code is single thread.

In muzero, I copy `games/gomoku.py` to `games/gobang.py` and changed some params.
changes:
```
board size from 11 to 6;
num in line from 5 to 4;
num simulations change to 400;
training steps change to 2000;
batch size change to 512;
played games per training step ratio set to 1;
```

full code see [gobang.py](https://github.com/zfavourite99/muzero-general/blob/master/games/gobang.py)

I trained about 47 hours. The result is still very poor. Trained on i7 7700 3.6G Hz 16G, RTX 2080 8G. And muzero implementation Multi-thread.

What went wrong? Can you help me?

My final goal is a 15x15 chessboard, but I want to train a simple one first. if it works, and then train a complex one. After all, the complex requires a lot of calculation.

Forgive me for my poor English, some content comes from google translation.

**Comment by ahainaut** [comment]
Hi,

Concerning Gomoku, this environment is a contribution, therefore we didn’t really focused on optimising the hyperparameters yet. 
Have you tried to tune the hyperparameters ? For example, you can change the parameters of ResNet, or increase the buffer size to allow more variations in the sampled games. If you already tried to tune the hyperparameters, could you send us the Tensorboard results so we can draw more precise conclusions ?

I pushed a version with new hyperparameters that seem more realistic.

About the comparison with AlphaZero, MuZero needs to learn a model, as it is designed for a more general setting. It requires training three neural networks instead of one as in AlphaZero. Moreover, the code you used to train AlphaZero takes advantage of the symmetries of the game and we don't. Such variations can explain the computational resources that are required to properly train MuZero.

Let us know about the results of your experiments !

**Comment by zfavourite99** [comment]
Thank you. I will conduct more tests later.
